File: crates/code-ranker-plugins/src/languages/go/tests/sample/mathx/mathx.go
// Package mathx provides small math helpers for the sample.
package mathx

import "example.com/sample/util"

// Square returns n*n, clamped to be at least zero via util.Max.
func Square(n int) int {
	v := n * n
	return util.Max(v, 0)
}

File: crates/code-ranker-plugins/src/languages/csharp/tests/sample/Mathx.cs
// Mathx.cs — depends on Sample.Util.
using Sample.Util;

namespace Sample.Mathx {
	public static class Squarer {
		// SquareClamped returns max(n*n, 0).
		public static int SquareClamped(int n) {
			int v = n * n;
			return Maths.Max(v, 0);
		}
	}
}
